selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/AddWorkspaceToOrganizationTest.java

```diff
@@ -0,0 +1,194 @@
+
+    // check that the Namespace link in workspace details correct
+    checkNamespaceLink(WS_NAME1, organization1.getName());
+    checkNamespaceLink(WS_NAME2, organization2.getName());
+    try {
+      checkNamespaceLink(WS_SUB_NAME1, suborgNameAdmin);
+    } catch (TimeoutException ex) {
+      // remove try-catch block after issue has been resolved
+      fail("Known issue https://github.com/eclipse/che/issues/7925");
+    }
+  }
+
+  @Test(priority = 1)
+  public void checkCreateWorkspaceAsMember() {
+    dashboard.logout();
+    dashboard.open(testUser.getName(), testUser.getPassword());
+
+    navigationBar.waitNavigationBar();
+    navigationBar.clickOnMenu(ORGANIZATIONS);
+    organizationListPage.waitForOrganizationsToolbar();
+    organizationListPage.waitForOrganizationsList();
+
+    organizationListPage.waitOrganizationInList(organization2.getName());
+
+    // create a workspace for organization2 and its suborganization
+    createWorkspace(organization2.getName(), WS_NAME3);
+    createWorkspace(suborgNameMember, WS_SUB_NAME2);
+
+    // check that workspace name exists in the Workspaces list and the Recent list
+    dashboard.selectWorkspacesItemOnDashboard();
+    workspaces.waitWorkspaceIsPresent(organization2.getName(), WS_NAME3);
+    workspaces.waitWorkspaceIsPresent(suborgNameMember, WS_SUB_NAME2);
+    assertTrue(dashboard.ifWorkspaceNamePresentInRecentList(WS_NAME3));
+    assertTrue(dashboard.ifWorkspaceNamePresentInRecentList(WS_SUB_NAME2));
+
+    // check that workspaces names of other users are not exist in the Recent list
+    assertFalse(dashboard.ifWorkspaceNamePresentInRecentList(WS_NAME1));
+    assertFalse(dashboard.ifWorkspaceNamePresentInRecentList(WS_SUB_NAME1));
+
+    // check that the Namespace link in workspace details correct
+    checkNamespaceLink(WS_NAME3, organization2.getName());
+
+    try {
+      checkNamespaceLink(WS_SUB_NAME2, suborgNameMember);
+    } catch (TimeoutException ex) {
+      // remove try-catch block after issue has been resolved
+      fail("Known issue https://github.com/eclipse/che/issues/7925");
+    }
+  }
+
+  private void createWorkspace(String organizationName, String workspaceName) {
+    dashboard.selectWorkspacesItemOnDashboard();
+    dashboard.waitToolbarTitleName("Workspaces");
+
+    workspaces.clickOnAddWorkspaceBtn();
+    newWorkspace.waitToolbar();
+    newWorkspace.openOrganizationsList();
+    newWorkspace.selectOrganizationFromList(organizationName);
+    newWorkspace.selectStack(JAVA.getId());
+    newWorkspace.typeWorkspaceName(workspaceName);
+    newWorkspace.clickOnCreateButtonAndEditWorkspace();
+
+    // TODO uncomment after https://github.com/eclipse/che/issues/8497 issue is fixed
+    // workspaceDetails.waitToolbarTitleName(workspaceName);
+
+    // timeout for workspaces list page to open
+    WaitUtils.sleepQuietly(2);
+  }
+
+  private void checkNamespaceLink(String workspaceName, String organizationName) {
```

Comment:
It better to have parameter sequence `checkNamespaceLink(String organizationName, String workspaceName)` which is similar to `createWorkspace(String organizationName, String workspaceName)` to avoid confusion.